google/cloud/forseti/common/gcp_api/errors.py

```diff
@@ -25,16 +40,16 @@ class ApiExecutionError(Error):
     CUSTOM_ERROR_MESSAGE = (
         'GCP API Error: unable to get {0} from GCP:\n{1}\n{2}')
 
-    def __init__(self, resource_name, e):
+    def __init__(self, api_execution_arg, e):
         """Initialize.
 
         Args:
-            resource_name (str): The resource name.
+            api_execution_arg (str): The resource name.
```

Comment:
Is resource name still the right description here?

Reply:
renamed back to resource_name